dataikuapi/dss/modelevaluationstore.py

```diff
@@ -110,12 +110,13 @@ def delete(self):
 
 
     ########################################################
-    # Runs
+    # Model evaluations
```

Comment:
(not part of this PR yet)

After quick check with @instanceofme , we want `project.list_model_evaluation_stores()` to return a list of objects instead of a list of dict by default. There is currently a ` as_type="object"` that does just that by it is not the default. The actual data of the ME should still be accessible using method's of the `DSSModelEvaluationStore`

Reply:
(I really mean we should get rid of "as_type")

Reply:
I put it out of consistency with other API. This is the only reason to me: I also see no value in getting dicts instead of objects as building those last ones is not long. So, if @instanceofme agrees to remove `as_type`, I will gladly remove it.

Reply:
He agrees, we discussed this orally

Reply:
ok, will update, thanks.

Reply:
Yes, `as_type` defaulting to dict was the old style, for new APIs we mostly do objects on which one may do `get_raw` to get the underlying dict if they want to.

Reply:
ok, done.